enterprise/internal/a8n/store.go

```diff
@@ -1326,6 +1335,13 @@ func listCampaignsQuery(opts *ListCampaignsOpts) *sqlf.Query {
 		preds = append(preds, sqlf.Sprintf("changeset_ids ? %s", opts.ChangesetID))
 	}
 
+	switch opts.State {
+	case CampaignStateOpen:
+		preds = append(preds, sqlf.Sprintf("closed_at is null"))
```

Comment:
Feel free to ignore me, but here's my last attempt to try to convert you to the one true way of writing SQL 😄 😎 

```suggestion
		preds = append(preds, sqlf.Sprintf("closed_at IS NULL"))
```

Reply:
Fine... :)